src/Driver/Mysqli/Connection.php

```diff
@@ -36,10 +36,19 @@ public function __construct(mysqli $connection)
      * Retrieves mysqli native resource handle.
      *
      * Could be used if part of your application is not using DBAL.
+     *
+     * @deprecated Call {@see getNativeConnection()} instead.
      */
     public function getWrappedResourceHandle(): mysqli
     {
-        return $this->connection;
+        Deprecation::trigger(
+            'doctrine/dbal',
+            'TODO',
```

Comment:
Let's not forget this (and others below)

Reply:
I keep falling into this trap. 🙈